LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Protocol Labs
```

Comment:
remove :) they did not work on this project

Reply:
Oh, yep.

Reply:
> remove :) they did not work on this project

done